tests/unit/scripts/test_importer.py

```diff
@@ -19,9 +19,33 @@
 # along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
 
 import pytest
+import os
 from scripts import importer
 
-_chrome_profile = 'tests/unit/scripts/chrome-profile'
+_samples = 'tests/unit/scripts/importer_sample'
+
+
+def qm_expected(input_format):
+    """Read expected quickmark-formatted output."""
+    with open(os.path.join(_samples, input_format, 'quickmarks'), 'r', encoding='utf-8') as f:
```

Comment:
Please break lines if longer than 79 chars, see [pylint output](https://travis-ci.org/qutebrowser/qutebrowser/jobs/302260842#L487).

Reply:
Sorry about that; for some reason it didn't show up on my local pylint run.